README.md

```diff
@@ -119,7 +120,7 @@ You can pass the following props to the `LiveChatLoaderProvider` provider:
   `2000`. Set to `0` to never load. This value is used in a `setTimeout` in
   browsers that don't support `requestIdleCallback`.
 
-## Providers
+## Supported Providers
 
 Currently there are two supported providers:
```

Comment:
I was wondering if we could use a `<details>` expander to list the provided examples? Then we could remove the "Currently there are two supported providers" label and shorten up the README a bunch. What do you think?

Reply:
👍 Good call. Updated Supported Providers to use `<details>` and put some templates for provider and component in `<details>` too.